projects/project-player-library/src/lib/constants/actionConstants.ts

```diff
@@ -25,6 +25,12 @@ export const actions = {
         action: "synced",
         color: "green"
     },
```

Comment:
check the spelling and is the icon relevant tocertificate?

Reply:
some of the icons are not rendering

Reply:
done